config/schedule.rb

```diff
@@ -4,16 +4,12 @@
 # We need Rake to use our own environment
 job_type :rake, "cd :path && govuk_setenv whitehall bundle exec rake :task --silent :output"
 
-def integration_or_staging?
-  ENV.fetch("GOVUK_WEBSITE_ROOT") =~ /integration|staging/
-end
-
 every :day, at: ["3am", "12:45pm"], roles: [:backend] do
   rake "export:mappings"
 end
 
 def search_index_consultations_cron_rule
-  if integration_or_staging?
+  if Whitehall.integration_or_staging?
```

Comment:
We should double check this on integration by deploying the branch, but I have a feeling that the `Whitehall` module might not be available when this file is set up on deploy...

Reply:
```
NameError: uninitialized constant #<Class:#<Whenever::JobList:0x00007f31ce8a6608>>::Whitehall
```

Seems like you're right. Will importing it work or is that not possible from here?